introduction/getting-started.qmd

```diff
@@ -8,15 +8,15 @@ source("scripts/before_chapter_script.R")
 ```
 
 
-This chapter serves as quick start guide for the code used throughout this book, and in particular the DeclareDesign package for the R programming language. DeclareDesign is a software implementation of every step of the declare-diagnose-redesign process. While you can declare, diagnose, and redesign using nearly any programming language, DeclareDesign is structured to make it easy to mix-and-match design elements while handling the tedious simulation bookkeeping behind the scenes. 
+This chapter serves as a quick start guide for the code used throughout this book, and in particular the `DeclareDesign` package for the R programming language. `DeclareDesign` is a software implementation of every step of the declare-diagnose-redesign process. While you can declare, diagnose, and redesign using nearly any programming language, `DeclareDesign` is structured to make it easy to mix and match design elements while handling the tedious simulation bookkeeping behind the scenes. 
 
-First, we provide instructions for getting started in R and RStudio. We then introduce the code structure for the three steps of the research planning process: declaration, diagnose, and redesign. After this introduction, readers should be able to *use* the code, but perhaps not write it themselves yet. We devote a longer section to getting started *writing* the code (@sec-ch13).
+First, we provide instructions for getting started in R and RStudio. We then introduce the code structure for the three steps of the research planning process: declaration, diagnosis and redesign. After this introduction, readers should be able to *use* the code, but perhaps not write it themselves yet. We devote a longer section (Chapter @sec-ch13) to getting started *writing* the code (@sec-ch13).
```

Comment:
Can you check this again? The edits include the chapter number twice now. Note (@sec-ch13) in Quarto will produce (Chapter 13).

Reply:
Fixed!